label_sync/repos.yaml

```diff
@@ -0,0 +1,242 @@
+Org: kubernetes
```

Comment:
It is used to help local development mode. You can save github points using this file instead of doing real Github API call.